spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -560,6 +560,33 @@
     end
   end
 
+  context "ansible tower job template accordion " do
+    before do
+      right_cell_text = nil
+      login_as user_with_feature(%w(providers_accord configured_systems_filter_accord configuration_scripts_accord))
+      controller.instance_variable_set(:@right_cell_text, right_cell_text)
```

Comment:
why not the simpler: `controller.instance_variable_set(:@right_cell_text, nil)`?